libcxx/test/std/thread/thread.jthread/cons.func.token.pass.cpp

```diff
@@ -46,7 +47,7 @@ int main(int, char**) {
   // Postconditions: get_id() != id() is true and ssource.stop_possible() is true
   // and *this represents the newly started thread.
   {
-    std::jthread jt{[] {}};
```

Comment:
Technically this test is specifically testing this constructor. If certain platforms does not support this constructor or need to pass in thread attributes, then it is no longer calling this constructor any more.  One could argue that this no longer tests the constructor as stated in the tests. But I am fine with this.

Reply:
I think we ended dropping the change for this exact reason!